docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -12,30 +12,24 @@ To install the PSW on Windows Server 2016 and Windows client < 1709, unpack the
 ZIP executable, and run the installer under `PSW_EXE_RS2_and_before`:
 
 ```cmd
-"C:\Intel SGX PSW for Windows v2.7.101.2\PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.7.101.2.exe"
+"C:\Intel SGX PSW for Windows v2.8.100.2\PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.8.100.2.exe"
 ```
 
 If you would like to manually update the PSW on Windows Server 2019 or Windows
-clients > 1709 without relying on Windows Update, you can do
-so as follows:
+clients > 1709 without relying on Windows Update, you can update the PSW components
+as follows:
 
-1. From the extracted PSW package, unzip the `PSW_INF_RS3_and_above\component\Signed_*.zip` file.
+- `devcon.exe` from the [Windows Driver Kit for Windows 10](
+  https://go.microsoft.com/fwlink/?linkid=2026156) can be used to update
+  the drivers from an elevated command prompt:
 
-   The following instructions will assume that it was unzipped to the `PSW_INF` folder.
+- Note that `devcon.exe` is usually installed to `C:\Program Files (x86)\Windows Kits\10\tools\x64`
+  which is not in the `PATH` environment variable by default.
 
-2. Install the PSW components:
-    - `devcon.exe` from the [Windows Driver Kit for Windows 10](
-      https://go.microsoft.com/fwlink/?linkid=2026156) can be used to update
-      the drivers from an elevated command prompt:
-
-    - Note that `devcon.exe` is usually installed to `C:\Program Files (x86)\Windows Kits\10\tools\x64`
-      which is not in the `PATH` environment variable by default.
-
-    - The hash values in the path may be different from the example command, please update as needed.
-
-    ```cmd
-    devcon.exe update PSW_INF\drivers\48e7c1e9-6de8-46ee-8ff9-46aa7b7ee5b9\sgx_psw.inf "SWC\VEN_INT&DEV_0E0C"
-    ```
+- From the `PSW_INF_RS3_and_above` folder, run the command:
+```cmd
+devcon.exe update sgx_psw.inf "SWC\VEN_INT&DEV_0E0C"
+```
```

Comment:
````suggestion

1. Install `devcon.exe`, available as part of the [Windows Driver Kit for Windows 10](
  https://go.microsoft.com/fwlink/?linkid=2026156).
    -  Note that `devcon.exe` is usually installed to `C:\Program Files (x86)\Windows Kits\10\tools\x64`
    which is not in the `PATH` environment variable by default.
2. In an elevated command prompt, run the following command from the extracted PSW package under the `PSW_INF_RS3_and_above` folder:
  ```cmd
  devcon.exe update sgx_psw.inf "SWC\VEN_INT&DEV_0E0C"
  ```
````

Reply:
Note that these changes also need to be mirrored to the WindowsManualSGX1Prereqs.md file as well.